takoyaki/src/main/java/frc/robot/subsystems/Climb.java

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.TalonFXInvertType;
+
+
+
+public class Climb extends SubsystemBase {
+
+    WPI_TalonFX left = new WPI_TalonFX(Constants.climbLeftCAN);
+    WPI_TalonFX right = new WPI_TalonFX(Constants.climbRightCAN);
+
+    Solenoid brakeSolenoid = new Solenoid(PneumaticsModuleType.CTREPCM, Constants.climbSolenoidChannel);
+
+    boolean brake = false;
+
+  public Climb() {
+    left.configFactoryDefault();
+    right.configFactoryDefault();
+
+    left.follow(right);
+
+    left.setInverted(TalonFXInvertType.OpposeMaster);
+    right.setInverted(TalonFXInvertType.Clockwise);
+  }
+
+  @Override
+  public void periodic() {
+    
+  }
+
+  @Override
+  public void simulationPeriodic() {
+  }
+
+  public void raiseClimbStart(){
```

Comment:
nit: one space before opening brace
```suggestion
  public void raiseClimbStart() {
```